Soccar [UI]/src/nl/socnet/message/ChangePlayerStatusMessage.java

```diff
@@ -0,0 +1,34 @@
+package nl.socnet.message;
+
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.ChangePlayerStatusMessageHandler;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.CHANGE_PLAYER_STATUS_MESSAGE_ID, handler = ChangePlayerStatusMessageHandler.class)
+public final class ChangePlayerStatusMessage extends Message {
+
+    private final Status status;
+
+    public ChangePlayerStatusMessage(Status status) {
+        this.status = status;
+    }
+
+    public Status getStatus() {
+        return status;
+    }
+
+    @Override
+    public int getId() {
+        return MessageConstants.CHANGE_PLAYER_STATUS_MESSAGE_ID;
+    }
+
+    public enum Status {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)